plugins/command-manager/src/main/java/com/wazuh/commandmanager/settings/PluginSettings.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package com.wazuh.commandmanager.settings;
+
+import com.wazuh.commandmanager.CommandManagerSettingsException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
+import org.opensearch.common.settings.KeyStoreWrapper;
+import org.opensearch.common.settings.SecureSettings;
+import org.opensearch.core.common.settings.SecureString;
+import org.opensearch.env.Environment;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+
+public class PluginSettings {
+    private static final Logger logger = (Logger) LogManager.getLogger(PluginSettings.class);
+
+    private static PluginSettings INSTANCE;
+
+    private static final String KEYSTORE_FILENAME = "wazuh-indexer.keystore";
+
+    private static KeyStoreWrapper keyStoreWrapper = KeyStoreWrapper.create();
+    private static Environment env;
```

Comment:
Link these properties to the PluginSettings instance, not to the class. For that, remove their static declaration and initialize them where required:

- Environment -> `+PluginsSettings::setEnvironment()`
- KeyStoreWrapper -> `-PluginsSettings::PluginsSettings()` (constructor)